### DISCUSSION TITLE
Update fastapi to support Pydantic2

### DISCUSSION THREAD

**Discussion by carnisj** [tiangolo/uvicorn-gunicorn-fastapi-docker#337] [Questions] [answered]
Hi, I'm using the python3.11 image with is built afaik with fastapi 0.88.0 currently. This doesn't support Pydantic2, would it be possible to use a more recent fastapi version?

**Comment by CharlesPerrotMinot** [comment]
That would be great @tiangolo

**Comment by tiangolo** [comment, marked as answer]
Now that Uvicorn supports managing workers with `--workers`, including restarting dead ones, there's no need for Gunicorn. That also means that it's much simpler to build a Docker image from scratch now, I updated the docs to explain it.

Because of that, I deprecated this Docker image: https://github.com/tiangolo/uvicorn-gunicorn-fastapi-docker#-warning-you-probably-dont-need-this-docker-image

Given that, you can start from an official recent Python Docker image, and install the latest FastAPI and the latest Pydantic. :sunglasses:
